Abha_Rana/my_profile/style.css

```diff
@@ -0,0 +1,69 @@
+body {
+    font: 20px Montserrat, sans-serif;
+    line-height: 1.8;
+    color: #f5f6f7;
+  }
+  p {font-size: 16px;}
+.margin { margin-bottom:10px;
+           margin-top:15px;
+           padding-top: 2%;
+           font-family: 'Shadows Into Light', cursive;
+        }
+.bg-1 { 
+      background-color: #1abc9c; /* Green */
+      color: #ffffff;
+    }
+    .bg-2 { 
+      background-color: #474e5d; /* Dark Blue */
+      color: #ffffff;
+    }
+    .bg-3 { 
+      background-color: #ffffff; /* White */
+      color: #555555;
+    }
+    .bg-4 { 
+      background-color: #2f2f2f; /* Black Gray */
+      color: #fff;
+    }
+.container-fluid {
+            padding-top: 25px;
+            padding-bottom: 60px;
+        
+          }
+ .navbar{
+            position:sticky;
+            padding-top: 5px;
```

Comment:
use shorthand for padding

Reply:
used padding instead of padding-top. issue fixed.